data/projects/coala_cli.md

```diff
@@ -1,24 +1,53 @@
-[cEP-0004](https://github.com/coala/cEPs/pull/29/files) was meant to improve
-the usability of coala. It is a very important proposal that never got the time
-and love it deserved.
+coala has come a long way from its early days in terms of its usability,
+athough all the changes that have been made are improvisations. A proper and
+planned overhaul of the user interface would help coala become much more user
+friendly and intuitive. At its core any software is as good as its interface
+with the user and this project aims to improve that.
+
+In this project we are going to focus on the command line interface (CLI) of
+coala. The specific areas and points to focus on are:
+
+1. Remove most of the text from the action selection screen.
+2. Use letters instead of numbers for selecting actions.
+3. Allow chaining actions in a single input.
+4. Add a 'select action for all results' operator.
+5. Make the implementation of ApplyPatchAction log the applied patch.
+6. Reprompt after running actions only if the action says so.
+7. Set 'Do nothing' as the default action.
+
+The end goal of the project is to make coala easier to use, hence the
+project is going to rely heavily upon heavy on mockups and feedback from the
+community. Also, in addition to the above changes the applicant is encouraged
+to come up with their own ideas to improve the CLI.
+
+[cEP-0004](https://github.com/coala/cEPs/pull/29/files) contains further
+details about the project. It is incomplete though and open to a lot of
+improvement.
 
-Goal of this project is to finish the cEP and implement usability improvements
-defined in it.
+#### MILESTONES
 
-Ideally a candidate comes up with own ideas that are not sketched in the cEP
-yet!
+##### PREPARATION/BONDING
 
-#### MILESTONES
+* The [cEP](https://coala.io/cep) has been updated, finished and merged with
+  the help of mockups and community feedback.
+* Preferably a survey is done on the importance of each proposed change to the
+  CLI.
+* The participant has a clear idea on how all parts of the cEP can be
+  implemented.
 
-##### GSOC 2017 COMMUNITY BONDING
+##### CODING PHASE 1
 
-* The cEP has been updated, finished and merged.
-* The participant has a clear idea on how all parts of the cEP can be implemented.
+* Implemented at least 3 improvements from the above list plus whatever has
+  been proposed by the participant.
+* Thorough testing of the implemented improvements is done and they are merged.
+* Documentation using illustrations of the improvements is merged.
 
-##### GSOC 2017 MIDTERM
+##### CODING PHASE 2
 
-* At least half of the points given in the cEP are done, merged, tested and properly documented.
+* Implemented, documented and tested at least 3 more improvements with coding
+  finished and merged.
```

Comment:
I guess thats ok. Milestones are not set in stone and not realy important for the org application. Google likes us having them but doesn't care too much